packages/devextreme/js/__internal/grids/new/grid_core/utils.test.ts

```diff
@@ -0,0 +1,68 @@
+import { describe, expect, it } from '@jest/globals';
+
+import { normalizeToolbarItems } from './utils';
+
+describe('normalizeToolbarItems', () => {
+  describe('when only default items are specified', () => {
+    it('should return correct items', () => {
```

Comment:
Looks like it's too much unspecific test names. It is not quite understandable what you mean by "should return correct items". It is okay to use it sometimes, but looks like here we can write something more specific